torch/ao/quantization/quantizer/x86_inductor_quantizer.py

```diff
@@ -28,14 +29,83 @@
     get_source_partitions,
     SourcePartition,
 )
-from .quantizer import QuantizationAnnotation, QuantizationSpec, Quantizer
+from .quantizer import (
```

Comment:
nit: probably better to import from "torch.ao.quantization.quantizer"

Reply:
Thanks for comment and changed.